ArcadiaParties-SPA/src/app/user/services/user.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { User } from '../models/User';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+  mockUser = {
+    identity: 'ekaterina.kuznetsova',
+    firstName: 'ekaterina',
+    lastName: 'kuznetsova',
+    department: 'horoshiy',
+    dates: [
+      { name: 'BirthDay', date: new Date('11.02.1944') },
+      { name: 'HireDay', date: new Date('03.12.1955') }
+    ],
+    roles: ['User', 'Admin']
+  } as User;
```

Comment:
I don't understand why do you use `as` keyword instead of `mockUser : User = {...}`

Reply:
It changed

Reply:
It is changed incorrectly. You removed type cast, but didn't add variable type.

Reply:
Variable type added

Reply:
Why did you add variable type, but remove method return type?